lib/confirmation.rb

```diff
@@ -0,0 +1,20 @@
+class Confirmation
+
+	attr_reader :checkout, :total
+
+	def initialize checkout
+		@checkout = checkout
+		@total = 0
+	end
+	
+	def user_cost user_total
+		total_cost
+		fail "The totals don't match, ours: £#{total} yours: £#{user_total}!" if total != user_total
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)